Feature/use bim data menu component

LrxGaelle opened the pull request:
https://platform-dev-use-bimdatamenu-component.bimdata.io

## Description
Use BIMDataMenu component

## Types of changes
<!--- Put an `x` in all the boxes that apply: -->
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist
<!--- Put an `x` in all the boxes that apply: -->
- [x] My code follows the code style of this project.
- [x] I have tested my code.

LrxGaelle committed:
bump design-system to 1.4.0-rc.89 version

LrxGaelle committed:
use BIMDataMenu component

NicolasRichel commented on src/components/specific/files/files-table/file-actions-cell/FileActionsCell.vue:
```diff
@@ -212,12 +136,137 @@ export default {
+      });
+    }
+
+    if (
```
This `if` statement refers to the exact same condition as the previous one.
I think we could merge them into one.

NicolasRichel replied:
**Note:** you can also use the [`hasAdminPerm` helper](https://github.com/bimdata/platform-next/blob/develop/src/utils/file-structure.js#L288) to simplify the code here. ;)

LrxGaelle committed:
some fix :

- add missing dataTestId,
- use $t in template and delete get text()
- refacto if condition

LrxGaelle committed:
fix versioning manager

LrxGaelle committed:
use iconEllipsisSize props on UserCardActionMenu

Bimdata-io commented:
:tada: This PR is included in version 1.0.0 :tada:

The release is available on [GitHub release](https://github.com/bimdata/platform/releases/tag/v1.0.0)

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:
